### PR TITLE
Add SSRF to the list of payouts for web bug bounty

### PR DESCRIPTION
**Pull request by fkiriakos07**
## Description
Add SSRF to the list of payouts for web bug bounty

### CODE REVIEW

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 1]
```diff
@@ -109,6 +115,7 @@ <h2 id="payouts-section">Payouts</h2>
     <li>For *.mozilla.org, *.mozilla.com, *.mozilla.net, and *.firefox.com.</li>
     <li>Lack of clickjacking protection (XFO, CSP) is insufficient to claim a bounty</li>
     <li>Requires a proof of concept to be eligible</li>
+    <li>SSRF must be proven to reach internal network hosts, pinging Burp Collaborator and performing DNS lookups are not enough to prove SSRF</li>
```
Comment (mozfreddyb) [inline bedrock/security/templates/security/web-bug-bounty.html:118]: Maybe emphasize **internal**.

We could also add a sentence *why* we don't consider pings and dns lookups enough (I assume there are middle-boxes and reverse proxies that sometimes do unavoidable requests which we don't care about? Maybe we could link to an e.g., akamai support page from here?)

Comment (fkiriakos07) [reply bedrock/security/templates/security/web-bug-bounty.html:118]: hmm not sure where to find a reference page for this

File: bedrock/security/templates/security/web-bug-bounty.html [hunk 2]
```diff
@@ -134,6 +141,7 @@ <h2 id="exclusions">Exclusions</h2>
     <li>Source code disclosures, as most of our code is open source</li>
     <li>Vulnerabilities discovered shortly after their public release</li>
     <li>Outdated TLS configurations which remain to support downloads from Windows XP systems</li>
+    <li>SSRF reports on services that are designed to load resources from the internet</li>
```
Comment (dveditz) [inline bedrock/security/templates/security/web-bug-bounty.html:144]: services that load things from the internet _might_ also have a legit SSRF problem, you just can't prove it using "Blind SSRF" probes. Is there a clearer way to word this? Maybe just that:

> "Blind SSRF" reports on services that [etc]

**Review by dveditz — CHANGES_REQUESTED** [review]
Looks good. I'd prefer one minor wording change to the exclusions section

File: bedrock/security/templates/security/web-bug-bounty.html (hunk 2 above)
Comment (fkiriakos07) [reply bedrock/security/templates/security/web-bug-bounty.html:144]: yes indeed, I can add that, thanks

**Review by dveditz — APPROVED** [review]
Looks good to me--thanks!

**Comment by fkiriakos07** [comment]
Hello @pmac , can you please review this PR for adding a new category of security bugs to our payouts list. I am not able to add reviewers to the PR, I think I don't have enough permissions on the repo.

**Comment by fkiriakos07** [comment]
Thanks @pmac I am doing a final check with the team to make sure we are ready to merge. In the meantime, how can I request more permissions on the repo to be able to add reviewers and labels in future PRs?

**Comment by fkiriakos07** [comment]
just confirmed that we are ready to merge the changes

**Comment by pmac** [comment]
I don't think GitHub separates those permissions from write permission to the repo unfortunately. I'll check again, but I think just mentioning one of us in the comments like you did works fine for now.

**Comment by fkiriakos07** [comment]
Hello @pmac , I see that pipeline for the merge commit failed and I don't see the changes deployed on bedrock, do I need to do anything to fix it?

**Comment by stevejalim** [comment]
Heya @fkiriakos07 - the CI fail when the branch was merged was down to an unrelated change (py2 syntax vs py3 syntax, and we needed to update some scripts to use `#!/usr/bin/env python3`)

A subsequent commit (8586533ccd, which fixed the above) passed CI and got deployed to dev, but that code hasn't flowed down to Staging or Prod just yet. You can check your that changes are up on the [Dev server](www-dev.allizom.org) if you want. They will hit prod with the next release. Let us know if you need those changes to hit live immediately.

**Comment by fkiriakos07** [comment]
thanks @stevejalim for the update, this is my very first commit so I am still learning the process, no rush to release the changes to prod, do you know when the next release will be?

**Comment by stevejalim** [comment]
Heya @fkiriakos07 - no worries at all! Looks like you might need permissions to see the CI progress in Gitlab, so worth Slacking with @pmac about that as he holds the keys. If you have any other questions, feel free to Slack me too.

Just checking if there's anything blocking doing a prod release now - will DM you

**Comment by pmac** [comment]
Prod pushes are manual but we usually do them a couple of times a week. If you have a date when the changes need to be live just let us know. 

Access to our pipelines on gitlab are not public and we don't allow access outside of the dev team for security reasons. But we can let you know when the next prod push happens.

**Comment by fkiriakos07** [comment]
I don't have a specific date in mind, the changes are not urgent so we don't mind going with your release schedule.

**Pull request merged**